lib/utils/Utils.dart

```diff
@@ -432,14 +431,9 @@ class Utils {
   }
 
   static Future<void> checkForUnfetchedData() async {
-    SharedPreferences prefs;
-    // if (userCreated["required"] && !userCreated["created"]) {
     final Map<String, bool> userCreated = initializedCheckData['userCreated'];
     if (userCreated['required']! && !userCreated['created']!) {
```

Comment:
(also remove the 'required' on the initalization of this dict)_
```suggestion
    if (!userCreated['created']!) {
```